BookingsApi/BookingsApi.DAL/Commands/CreateVideoHearingCommand.cs

```diff
@@ -104,6 +102,7 @@ public async Task Handle(CreateVideoHearingCommand command)
             }
             
             videoHearing.UpdateBookingStatusJudgeRequirement();
+            await _context.VideoHearings.AddAsync(videoHearing);
```

Comment:
is there a reason for moving this line? I mean it makes sense to add to the context add the end before calling SaveChanges(), just wondering if there was something that prompted it

Reply:
Yeah I can move it back actually, forgot I changed it. Was having andissue with the properties not updating in the db when saving the context and was trying to figure it out what it was and moved that. Turns out it was something else unrelated. Will set it back how it was.